src/ethereum/EtherscanButton/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Icon } from '../..';
+
+type Props = {
+  className?: string;
+  network?: 'mainnet' | 'rinkeby';
+  type: 'address' | 'tx';
+  value: string;
+};
+
+const EtherscanButton = ({
+  className,
+  type,
+  value,
+  network = 'mainnet',
+}: Props): React.ReactElement => {
+  const getNetwork = () => {
```

Comment:
this could be defined outside the component

Reply:
done!